crates/ruff_linter/resources/test/fixtures/pyflakes/F822_3.py

```diff
@@ -0,0 +1,7 @@
+"""Respect `# noqa` directives on `__all__` definitions."""
+
+__all__ = [  # noqa: F822
+    "Bernoulli",
+    "Beta",
+    "Binomial",
+]
```

Comment:
Is it worth also adding this to the test, to illustrate that if you _do_ only `noqa` one of the items in `__all__`, you'll still get an `F822` error on the other items (if they're also undefined)? That seems desirable to me, so I think probably worth asserting?

```suggestion
]

if True:
    __all__ += [
        "ContinuousBernoulli",  # noqa: F822
        "ExponentialFamily",  # F822 emitted here
    ]

```